### DISCUSSION TITLE
10 Vulnerabilities?

### DISCUSSION THREAD

**Discussion by Overloader6** [nasa/openmct#4174] [Community Q&A] [answered]
![image](https://user-images.githubusercontent.com/22007229/131754080-f58d3c3f-d6b4-4ca5-afbe-e21ace9056ed.png)
As you can see above npm is reporting 10 vulnerabilitites upon `npm install`. Should I be concerned?

**Comment by akhenry** [comment, marked as answer]
Thanks for reporting. These vulnerabilities all relate to dependencies of our build and test tooling. None of them are present in Open MCT at runtime.

We will update some of our build time dependencies to address these, but rest assured these are not vulnerabilities in Open MCT itself. 

It's worth mentioning that we do use dedicated static analysis tools for identifying security vulnerabilities in our runtime code.

**Comment by unlikelyzero** [comment]
@Voreck please follow our progress here: https://github.com/nasa/openmct/issues/2449
